### PR TITLE
The first letter should be capitalized

### PR DESCRIPTION
**Pull request by yu8833**

### CODE REVIEW

File: docs/examples/couchbase.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ curl -v -X POST http://192.168.99.100:8091/pools/default -d memoryQuota=300 -d i
 > Content-Length: 36
 > Content-Type: application/x-www-form-urlencoded
 >
-* upload completely sent off: 36 out of 36 bytes
+* Upload completely sent off: 36 out of 36 bytes
```
Comment (vdemeester) [inline docs/examples/couchbase.md]: This one should definitely not be updated, it's the output of the command 👼

**Comment by YuPengZTE** [comment]
Thanks @GordonTheTurtle. It is sorry that the develop branch "The-first-letter-should-be-captialed" is deleted, is there any way to correct this mistake in this PR?

File: docs/getstarted/index.md [hunk 2]
```diff
@@ -36,13 +36,13 @@ If you have already downloaded and installed Docker, you are ready to run Docker
...
 You'll learn how to:
 
-* install Docker software for your platform
-* run a software image in a container
-* browse for an image on Docker Hub
-* create your own image and run it in a container
-* create a Docker Hub account and an image repository
-* create an image of your own
-* push your image to Docker Hub for others to use
+* Install Docker software for your platform
```
Comment (sfsmithcha) [inline docs/getstarted/index.md]: these bullet points are the continuation of the sentence above:

You'll learn how to _install_ ... . They do not require capitalization.

**Comment by sfsmithcha** [comment]
Thank you for your contribution @YuPengZTE . I'm closing this for a couple of reasons:

1. Output should reflect the CI output per @vdemeester comment
2. Capitalization is not required in the case of these bullets.

**Comment by YuPengZTE** [comment]
Thanks @sfsmithcha, I close this PR.

**Pull request closed without merging**